### PR TITLE
Docs on package scopes

### PR DESCRIPTION
**Pull request by tylerbutler**

### CODE REVIEW

File: docs/content/docs/build/packages.md [hunk 1]
```diff
@@ -1,14 +1,72 @@
...
+need two or three of these packages for typical Fluid development: The `fluid-framework` package, which contains the
+public API for the Fluid Framework, and a service-specific client package, such as the
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
```
Comment (heliocliu) [inline docs/content/docs/build/packages.md:15]: i thought it was a bit odd that this sentence was basically the same as the previous one, but used a different example.  may be better to use the same example and link to the available service clients page

File: docs/content/docs/build/packages.md [hunk 2]
```diff
@@ -1,14 +1,72 @@
...
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
+
+### IFluidContainer
+
+The [IFluidContainer][] interface is the one of
+the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`. It
+includes functionality to retrieve the Fluid data contained within a container, as well as to inspect the state of the
```
Comment (heliocliu) [inline docs/content/docs/build/packages.md:23]: how well understood is the container concept expected to be at this point?  thinking if "within the container" or "within itself" (or something else) might make more sense here.  someone unaware of the underlying container bit might be wondering about that relation

File: docs/content/docs/build/packages.md [hunk 3]
```diff
@@ -1,14 +1,72 @@
...
+## Package scopes
+
+Fluid Framework packages are published under one of the following npm scopes:
+
+- @fluidframework
+- @fluid-experimental
+- @fluid-internal
+- @fluid-tools
+
+Unless you are contributing to the Fluid Framework, you should only need packages from the **@fluidframework** scope.
```
Comment (heliocliu) [inline docs/content/docs/build/packages.md:44]: is it worth calling out `fluid-framework` here which is scopeless?

**Review by heliocliu — APPROVED** [review]

File: docs/content/docs/build/packages.md [hunk 4]
```diff
@@ -1,14 +1,72 @@
...
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
+
+### IFluidContainer
```
Comment (skylerjokiel) [inline docs/content/docs/build/packages.md:19]: I'm concerned about the implications of talking about the interface vs the object. The main problem I have is that talking about interface means talk about Typescript. This is not inclusive to the JavaScript community and honestly confusing IMO. I know Matt updated the readme to support this syntax but I feel that talking about interfaces are to OO of a concept for JS devs.

Comment (skylerjokiel) [reply docs/content/docs/build/packages.md:19]: I would rather this section talk about `FluidContainer`

File: docs/content/docs/build/packages.md [hunk 5]
```diff
@@ -1,14 +1,72 @@
...
+
+The [IFluidContainer][] interface is the one of
+the types returned by calls to `createContainer()` and `getContainer()` on the service clients such as `AzureClient`. It
+includes functionality to retrieve the Fluid data contained within a container, as well as to inspect the state of the
+collaboration session connection.
+
+### DDS packages
+
+You'll use one or more DDSes in your container to model your collaborative data.  The `fluid-framework` package includes
+three data structures that cover a broad range of scenarios:
```
Comment (skylerjokiel) [inline docs/content/docs/build/packages.md:29]: Let's just use the generic shared object term.

```suggestion
### Shared object packages

You'll use one or more shared objects in your container to model your collaborative data.  The `fluid-framework` package includes
three data structures that cover a broad range of scenarios:
```

File: docs/content/docs/build/packages.md (hunk 3 above)
Comment (skylerjokiel) [reply docs/content/docs/build/packages.md:44]: and `tinylicious` service

File: docs/content/docs/build/packages.md [hunk 6]
```diff
@@ -1,14 +1,72 @@
...
+[IFluidContainer]: {{< relref "ifluidcontainer.md" >}}
+[PureDataObject]: {{< relref "puredataobject.md" >}}
+[PureDataObjectFactory]: {{< relref "puredataobjectfactory.md" >}}
+[SharedCounter]: {{< relref "/docs/data-structures/counter.md" >}}
+[SharedMap]: {{< relref "/docs/data-structures/map.md" >}}
+[SharedNumberSequence]: {{< relref "sequences.md#sharedobjectsequence-and-sharednumbersequence" >}}
+[SharedObjectSequence]: {{< relref "sequences.md#sharedobjectsequence-and-sharednumbersequence" >}}
+[SharedSequence]: {{< relref "sequences.md" >}}
+[SharedString]: {{< relref "string.md" >}}
+[TaskManager]: {{< relref "/docs/data-structures/task-manager.md" >}}
```
Comment (skylerjokiel) [inline docs/content/docs/build/packages.md:70]: What's this auto-generated from? Most of these things are not included in the fluid-framework package

File: docs/content/docs/build/packages.md (hunk 1 above)
Comment (skylerjokiel) [reply docs/content/docs/build/packages.md:15]: Agreed. I think we can just axe the first sentence. GitHub is being odd about suggestions atm so I can't select it.

**Review by skylerjokiel — COMMENTED** [review]
Some nits but overall really liking the direction

File: docs/content/docs/build/packages.md (hunk 4 above)
Comment (tylerbutler) [reply docs/content/docs/build/packages.md:19]: Where do I see the FluidContainer type show up? I.e. is there any time where I am going to get back a FluidContainer as opposed to an IFluidContainer?

File: docs/content/docs/build/packages.md (hunk 6 above)
Comment (tylerbutler) [reply docs/content/docs/build/packages.md:70]: See the reference at the top of the links section. These are "global links."

File: docs/content/docs/build/packages.md [hunk 7]
```diff
@@ -1,14 +1,72 @@
...
+`@fluidframework/tinylicious-client` package.
 
-Experimental scope and packages.
+## Primary API: fluid-framework
 
-List other scopes and their meaning (or link to the wiki).
+The `fluid-framework` package bundles a collection of Fluid Framework client packages for easy use when paired with a
+corresponding service client package, such as the `@fluidframework/azure-client` package. The `fluid-framework` package
+consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures
+(DDSes).
```
Comment (tylerbutler) [inline docs/content/docs/build/packages.md:17]:
```suggestion
The `fluid-framework` package consists primarily of two portions: the `IFluidContainer` interface and a selection of distributed data structures (DDSes).
```

File: docs/content/docs/build/packages.md (hunk 2 above)
Comment (tylerbutler) [reply docs/content/docs/build/packages.md:23]:
```suggestion
includes functionality to retrieve the Fluid data contained within itself, as well as to inspect the state of the
```

File: docs/content/docs/build/packages.md (hunk 3 above)
Comment (tylerbutler) [inline docs/content/docs/build/packages.md:44]:
```suggestion

In addition to the scoped packages, two unscoped packages are published: the `fluid-framework` package, described earlier, and the `tinylicious` package, which contains a minimal Fluid server. For more information, see [Tinylicious]({{< relref "tinylicious.md" >}}

Unless you are contributing to the Fluid Framework, you should only need the unscoped packages and packages from the **@fluidframework** scope.
```

**Pull request merged**